app/controllers/user_controller.rb

```diff
@@ -0,0 +1,44 @@
+module TheApp
+
+  module Routes
+
+    class UserController < Base
+      PERMITTED_PARAMS = [:email, :username, :name, :password, :password_confirmation]
+
+      before do
+        params.each do |k, v|
+          self.class.send(:define_method, k){v} if PERMITTED_PARAMS.include?(k.to_sym)
```

Comment:
Line is too long. [86/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.